### DISCUSSION TITLE
Random Timers...

### DISCUSSION THREAD

**Discussion by Gamerzoe** [runelite/runelite#17509] [Q&A] [answered]
Occasionally, I have had random timers pop up with various icons and locations (mostly priff/rimm). Sometimes its fruit, trees, books, etc. I have been to these areas and found nothing out of the ordinary and it doesn't seem to be triggered by any particular action. It just shows an icon, place, and countdown. Has anyone experienced something like this? I've combed through plugins to see if this was a setting and came out dry. 
![Random Timer Osrs](https://github.com/runelite/runelite/assets/123108091/6d63aa71-da2c-4d30-b689-4e8a3af8d65e)

**Comment by cdfisher** [comment, marked as answer]
This is a feature of the "Forestry CC" hub plugin and is not related to the core client. Disable that and it should go away.

**Reply by Gamerzoe** [reply]
you are a life saver thank you
